app/javascript/stylesheets/alert.scss

```diff
@@ -0,0 +1,11 @@
+// Dismissible alerts
+
+// Expand the left padding and account for the close button's positioning.
+.alert-dismissible {
+  padding-left: $alert-dismissible-padding-r;
+
+  // Move alert button close link position to left
+  .btn-close {
+    left: 0;
+  }
+}
```

Comment:
by default, bootstrap puts the alert close button on the RIGHT.  This can be far away from the alert text, if the window is wide and/or if the alert text is short.   It looked like the access button class used puts the button on the LEFT, so this is the simple change to use the correct bootstrap button and put it on the LEFT.

Reply:
Sounds good to me.